pulsar/producer.go

```diff
@@ -128,6 +128,8 @@ type Producer interface {
 	Send(context.Context, *ProducerMessage) (MessageID, error)
 
 	// SendAsync a message in asynchronous mode
+	// This call will be blocking when the `event channel` becomes full (default: 10) or the
```

Comment:
```suggestion
	// This call is blocked when the `event channel` becomes full (default: 10) or the
```

What's the unit for "10"? 10 MB or  10 messages?